### PR TITLE
Laboratory work #1, Alyona Gosteva - 21FPL2

### PR DESCRIPTION
**Pull request by Gostaa**

### CODE REVIEW

File: lab_1_keywords_tfidf/main.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Lab 1
 Extract keywords based on frequency related metrics
 """
+# testing change
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:5]: remove

File: seminars/practice_1_intro.py [hunk 2]
```diff
@@ -32,11 +32,32 @@
...
-
 # Case 2: the second debugging case:
 if first_num * second_num == third_num:
     print('First number * second equals third number')
 else:
     print('First number * second does not equal third number')
 
 print('Program finished')
+
+example = 'Hello'
```
Comment (artyomtugaryov) [inline seminars/practice_1_intro.py:43]: please, do not commit these changes. Only labs

**Review by artyomtugaryov — COMMENTED** [review]
Hello, @Gostaa! As a reminder, the deadline for the first lab is September 30th and you have less than 2 weeks to complete it. Please hurry to be on time.

File: seminars/practice_1_intro.py (hunk 2 above)
Comment (Gostaa) [reply seminars/practice_1_intro.py:43]: is there a way to delete these changes?

File: lab_1_keywords_tfidf/main.py (hunk 1 above)
Comment (Gostaa) [reply lab_1_keywords_tfidf/main.py]: done

File: lab_1_keywords_tfidf/main.py [hunk 3]
```diff
@@ -17,7 +18,18 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+    else:
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:23]: Remove `else:` - https://tutorial.eyehunts.com/python/python-one-line-if-without-else-example-code/

File: lab_1_keywords_tfidf/main.py [hunk 4]
```diff
@@ -33,7 +45,15 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    text_unstopworded = []
+    if not (isinstance(tokens, list) and isinstance(stop_words, list)):
+        return None
+    else:
```
Comment (artyomtugaryov) [inline lab_1_keywords_tfidf/main.py:51]: again

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Please, remove changes in the `seminars` folder

**Pull request closed without merging**